### PR TITLE
Chapter 4

### PR DESCRIPTION
**Pull request by thuongnn-0898**

### CODE REVIEW

**Comment by thuongnn-0898** [comment]
ready

**Comment by VuThiTranVan** [comment]
commented
2 commit chú ơi

**Comment by thuongnn-0898** [comment]
ready

**Comment by thuongnn-0898** [comment]
ready

**Comment by thuongnn-0898** [comment]
ready

File: app/helpers/static_pages_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module StaticPagesHelper
+end
```
Comment (VuThiTranVan) [inline app/helpers/static_pages_helper.rb]: Xóa file hì

**Comment by VuThiTranVan** [comment]
merged
fix comment rồi mợt nhé

**Pull request merged**